doc/devel/coding_guide.rst

```diff
@@ -13,9 +13,89 @@
 Pull request guidelines
 ***********************
 
-Pull requests (PRs) are the mechanism for contributing to Matplotlibs code and
+Pull requests (PRs) are the mechanism for contributing to Matplotlib's code and
```

Comment:
```suggestion
Pull requests (PRs) on GithHub are the mechanism for contributing to Matplotlib's code and
```

Worth specifying where?

Reply:
Maybe link "Pull requests" to https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests